Documentation/prereqs.md

```diff
@@ -0,0 +1,51 @@
+.NET Core native prerequisites
+==============================
+
+This document outlines the dependencies needed to run .NET Core tools and applications. 
+
+## Windows dependencies
```

Comment:
https://github.com/dotnet/core/blob/master/roadmap.md here, I will link, that's a good point.